Tests/UnitTests/EdgeHitProcessorTests.swift

```diff
@@ -46,17 +47,37 @@ class EdgeHitProcessorTests: XCTestCase {
             return SharedStateResult(status: .set, value: [ASSURANCE_INTEGRATION_ID: "test-int-id"])
         }
 
-        if extensionName == IDENTITY_SHARED_STATE {
-            return SharedStateResult(status: .set, value: [IDENTITY_SHARED_STATE: "test-mcid"])
-        }
-
         if extensionName == CONFIGURATION_SHARED_STATE {
             return SharedStateResult(status: .set, value: [EDGE_CONFIG_ID: "test-config-id"])
         }
 
         return nil
     }
 
+    private func resolveXDMSharedState(extensionName: String, event: Event?) -> SharedStateResult? {
+        if extensionName == IDENTITY_SHARED_STATE {
+            guard let identityMapData = """
+                {
+                  "ECID" : [
+                    {
+                      "authenticationState" : "ambiguous",
+                      "id" : "test-mcid",
+                      "primary" : false
+                    }
+                  ]
+                }
```

Comment:
The IdentityEdge shared state contains a single key "identityMap" which then contains all the identifiers, so the data should look like:
```
{
  "identityMap" : {
    "ECID" : [
      {
        "authenticationState" : "ambiguous",
        "id" : "test-mcid",
        "primary" : false
      }
    ]
  }
}
```

Reply:
In this case, you're correct. I'll update it.